src/features/onboarding/AccountAssignScreen.tsx

```diff
@@ -148,15 +136,14 @@ const AccountAssignScreen = () => {
       }
 
       if (hasAccounts) {
-        rootNav.reset({
-          index: 0,
-          routes: [{ name: 'ServiceSheetNavigator' }],
-        })
-        reset()
+        close()
+        // wait 1 second
+        await new Promise((resolve) => setTimeout(resolve, 1000))
```

Comment:
nit: you should make a `sleep` function. But also why do we need to wait.

Reply:
Because bip39 and keypair generation functions stall the whole UI thread. I wait so we can show some loading animation while keypair generation does its thing